src/platform/site-wide/index.js

```diff
@@ -15,7 +15,7 @@ import startAnnouncementWidget from './announcements';
 import startVAFooter from './va-footer';
 import addFocusBehaviorToCrisisLineModal from './accessible-VCL-modal';
 
-import(/* webpackChunkName: "uswds" */ 'uswds');
+import('@department-of-veterans-affairs/formation/dist/formation');
```

Comment:
Picky but I think we can remove the parentheses here to be more consistent with the rest of our imports